From 9f480fcbc9cb7bfb0d0bde8727dbfba9d878b596 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Christian=20Gr=C3=BCnhage?=
 <jan.christian@gruenhage.xyz>
Date: Sun, 4 Sep 2022 14:38:10 +0200
Subject: [PATCH] Fix values in test cases

SeedableRng values are "stable", but minor version bumps (which have
happened since these cases were written) are allowed to change those. A
breaking change here was documentated for 0.7.
---
 tests/tests.rs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/tests.rs b/tests/tests.rs
index d569663..ed4eff4 100644
--- a/tests/tests.rs
+++ b/tests/tests.rs
@@ -33,7 +33,7 @@ fn make_reinhold_vector() -> Vec<ReinholdWord> {
 
 #[test]
 fn beale_rng_test() {
-    let wanted: Vec<BealeWord> = vec!["dr", "raced", "pvc", "moon"]
+    let wanted: Vec<BealeWord> = vec!["io", "gavel", "beam", "time"]
         .into_iter()
         .map(BealeWord::new)
         .collect();
@@ -45,7 +45,7 @@ fn beale_rng_test() {
 
 #[test]
 fn reinhold_rng_test() {
-    let wanted: Vec<ReinholdWord> = vec!["douse", "qo", "prune", "moan"]
+    let wanted: Vec<ReinholdWord> = vec!["india", "gamma", "bcd", "theme"]
         .into_iter()
         .map(ReinholdWord::new)
         .collect();
-- 
2.37.3

